src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -27,8 +27,7 @@ def list(self, subsystem_nqn: str):
         @CreatePermission
         def create(self, rbd_pool: str, rbd_image: str, subsystem_nqn: str,
                    create_image: Optional[bool] = True, image_size: Optional[int] = 1024,
-                   block_size: int = 512, nsid: Optional[int] = 1,
-                   uuid: Optional[str] = None, anagrpid: Optional[int] = 1):
+                   block_size: int = 512):
```

Comment:
I'm a bit concerned about having to maintain these defaults in sync with ceph-nvmeof. Perhaps, rather than using specific values (`1024`, `512`), we could set those by default to `None`, and if these are `None` we don't pass those:
```py
def create(self, rbd_pool: str, rbd_image: str, subsystem_nqn: str,
                   create_image: Optional[bool] = None,
                   image_size: Optional[int] = None,
                   block_size: Optional[int] = None):
    optional = dict(
        create_image=create_image,
        image_size=image_size,
        block_size=block_size,
    )  
    response = NVMeoFClient().create_namespace(rbd_pool, rbd_image, subsystem_nqn, 
                                           **{k:v if k,v in optional.items() if v is not None}
```
In any case, this results in more obscure code, so let's take it just as a caution.

Reply:
yeah, i didn't wanted to put much calculations in this area and went with a bit more simpler one. and yes, we can't rely on the defaults of nvme-of projects because they are subject to change. do you want me to make these changes? i suppose i'll have to do something similar to this in a couple of other apis as well